app/src/main/java/io/left/meshim/models/Message.java

```diff
@@ -253,4 +237,23 @@ public void writeToParcel(Parcel dest, int flags) {
         dest.writeInt(recipientId);
         dest.writeByte((byte) (isMyMessage ? 1 : 0));
     }
+
+    /**
+     * Function to format the date as desired
+     * @param date needing formatting
+     * @return  the formatted date as a string
+     */
+    public static String dateFormat(Date date){
```

Comment:
`formatDate()` would make this more clear that it's a function (verbNoun). The property was `dateFormat` before because it was an attribute of type `DateFormat`.